app/javascript/utilities/viewport.js

```diff
@@ -3,22 +3,45 @@
  *
  * @example
  * const element = document.getElementById('element');
- * isInViewport(element); // true or false
+ * isInViewport({element, allowPartialVisibility = true}); // true or false
  *
  * @param {object} element - The HTML element to check
  * @param {number} [offsetTop=0] - Part of the screen to ignore counting from the top
- *
+ * @param {boolean} [allowPartialVisibility=false] - A boolean to flip the check between partial or completely visible in the viewport
  * @returns {boolean} isInViewport - true if the element is visible in the viewport
  */
-export function isInViewport(element, offsetTop = 0) {
+export function isInViewport({
+  element,
+  offsetTop = 0,
+  allowPartialVisibility = false,
+} = {}) {
```

Comment:
(refactor/blocking) I don't think we need the default empty object here - we don't want the function to be called without at least the `element` property, so we can safely expect an object passed in

```suggestion
}) {
```

Reply:
> Thanks for making those changes so quickly! Just one small thing left.
> 
> Don't forget to also sign the CLA linked above 🙂

@aitchiss Hey! 👋  I signed the CLA, but it isn't picking up. Not sure what is missed. Let me try again.